### PR TITLE
pools: Add mStable pool

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: contracts/pools/MstableTempusPool.sol [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        )
+    {
+        savingsContract = _savingsContract;
+    }
+
+    function depositToUnderlying(uint256 amount) internal override returns (uint256) {
+        require(msg.value == 0, "ETH deposits not supported");
+
+        IERC20(backingToken).safeIncreaseAllowance(address(savingsContract), amount);
+        return savingsContract.depositSavings(amount);
```
Comment (RedFox20) [inline contracts/pools/MstableTempusPool.sol:50]: Do we trust the return value here?

File: contracts/pools/MstableTempusPool.sol [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        override
+        returns (uint256 backingTokenAmount)
+    {
+        backingTokenAmount = savingsContract.redeemCredits(yieldBearingTokensAmount);
+        IERC20(backingToken).transfer(recipient, backingTokenAmount);
+    }
+
+    /// @return Updated current Interest Rate with the same precision as the BackingToken
+    function updateInterestRate() internal view override returns (uint256) {
+        // TODO: it doesn't have a public update function
```
Comment (RedFox20) [inline contracts/pools/MstableTempusPool.sol:64]: That's fine, not all protocols need it, only some which modify state. So this implementation looks correct

**Comment by RedFox20** [comment]
This needs: 

- [ ] support in TempusPool.ts
- [ ] support in MultiPoolTestSuite.ts
- [ ] support in Config.ts
- [ ] Separate implemention of integration tests

**Comment by axic** [comment]
It also needs a mock implementation.

**Comment by mijovic** [comment]
Closing, as we don't have a plan to finish this integration soon

**Pull request closed without merging**